client/hosting/server-settings/components/defensive-mode-card/index.tsx

```diff
@@ -100,16 +102,52 @@ export default function DefensiveModeCard( { disabled }: DefensiveModeCardProps
 						</HostingCardDescription>
 					</div>
 
-					<Button
-						className="defensive-mode-card__button"
-						busy={ isEnabling }
-						disabled={ disabled || isEnabling }
-						onClick={ () => {
-							mutate( { active: false } );
-						} }
-					>
-						{ translate( 'Disable defensive mode' ) }
-					</Button>
+					{ ! defensiveModeData.enabled_by_a11n && (
+						<Button
+							className="defensive-mode-card__button"
+							busy={ isEnabling }
+							disabled={ disabled || isEnabling }
+							onClick={ () => {
+								mutate( { active: false } );
+							} }
+						>
+							{ translate( 'Disable defensive mode' ) }
```

Comment:
Do you think it would be helpful to add comments to those new strings to explain to translators what exactly _defensive mode_ is?

Reply:
Excellent point 👍